README.md

```diff
@@ -3,9 +3,12 @@
 # Ardite Core
 
 [![Build Status][1]][2]
+[![Clippy Linting Result][3]][4]
 
 [1]: https://travis-ci.org/ardite/ardite-core.svg?branch=master
```

Comment:
I like that you do it this way, but maybe place the links at the bottom? Sort of more like a bib of sorts.